apps/keyboard/js/settings/word_list_converter.js

```diff
@@ -405,7 +407,7 @@ TSTBlobBuilder.prototype.toBlobArray = function() {
     this._emitNode(node);
   }, this);
 
-  return this._output;
+  return this._outputBuffer;
```

Comment:
Actually...I can still just instantiate a Uint8Array without explicitly instantiating an ArrayBuffer and just return ``this._output.buffer`` in ``toBlobArray()``. This approach is *much* cleaner and eliminates most code changes.